plugins/prefixIds.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+exports.type = 'perItem';
+
+exports.active = false;
+
+exports.params = {
+    delim: '__'
+};
+
+exports.description = 'prefix IDs';
+
+
+var path      = require('path'),
```

Comment:
hmm, forgot to say sorry, but on my side, I'm more using svgo from browser, and I try to remove those, node-only deps. But it shouldn't be hard to pass a prefix string, not a path